linode/manager
# upcoming: [M3-8364] - Use React Hook Form instead of Formik for Create Bucket

## Description 📝
Switching from Formik to React Hook Form before updating the drawer with the new UI / API requirements.

## Changes  🔄
- Using `isFeatureEnabledV2` instead of deprecated `isFeatureEnabled`
- Unique label validation is happening in the schema now
- Reset of values is now happening `onExited` 

## Target release date 🗓️
N/A

## Preview 📷
N/A

## How to test 🧪

>[!note]
Ensure you test both legacy and multi-cluster (gen1) by toggling flag

### Verification steps
- Create a new bucket and ensure the creation process still works
- Try creating a bucket with a label that's already used and ensure the validation works
   - You should see an error with helper text under label input field 
- The bucket creation form should be more responsive, with real-time validation feedback
- Form fields should validate input as the user types, not just on submission
- The form should prevent submission if there are any validation errors

## As an Author I have considered 🤔

*Check all that apply*

- [  ] 🧪 Providing/Improving test coverage
